### ISSUE TITLE
Ember cannot build without a travis.yml file

### ISSUE THREAD

**Issue by devotox** [ember-cli/ember-cli#9025]
Output from `ember version --verbose && npm --version && yarn --version`:
```
ember-cli: 3.15.2
node: 12.15.0
v8: 7.7.299.13-node.16
uv: 1.33.1
zlib: 1.2.11
brotli: 1.0.7
ares: 1.15.0
modules: 72
nghttp2: 1.40.0
napi: 5
llhttp: 2.0.4
http_parser: 2.9.3
openssl: 1.1.1d
cldr: 35.1
icu: 64.2
tz: 2019c
unicode: 12.1
os: darwin x64
6.13.4
1.22.0
```

Moving from Travis to Github Actions as arisen an issue in platform_checker [Platform Checker Line 14](https://github.com/ember-cli/ember-cli/blob/master/lib/utilities/platform-checker.js#L14)

### Steps to Reproduce
1. Delete `.travis.yml`
2. Run `ember build`
3. Don't Profit

**Comment by jherdman** [comment]
I cannot confirm this with an app I have on 3.15.2

```
ember-cli: 3.15.2
http_parser: 2.8.0
node: 8.16.2
v8: 6.2.414.78
uv: 1.23.2
zlib: 1.2.11
ares: 1.10.1-DEV
modules: 57
nghttp2: 1.39.2
napi: 4
openssl: 1.0.2s
icu: 60.1
unicode: 10.0
cldr: 32.0
tz: 2017c
os: linux x64
```

**Comment by devotox** [comment]
I was attempting this with an addon which may be where the issue lies?

**Comment by jherdman** [comment]
Hm... I upgraded my [imgix-photo](https://github.com/PrecisionNutrition/imgix-photo) addon and didn't see a problem either.

**Comment by devotox** [comment]
and you ran `ember build` and `ember build --prod`

**Comment by jherdman** [comment]
Confirm.

**Comment by devotox** [comment]
Was that a confirmation that you encountered the error or you ran that and no error. If so I can try and see what happens if I create a brand new addon

**Comment by jherdman** [comment]
My apologies, I confirm that neither `ember build`, or `ember build --prod` result in an error.

**Comment by devotox** [comment]
Ok, thank you. I will try and get more information for you about an exact test case 🙏

**Comment by rwjblue** [comment]
@devotox the platform checker code you referenced is checking for `.travis.yml` in ember-cli's own node_modules directory, not the project's. Did you perhaps delete _all_ `.travis.yml` files (e.g. recursively)?

**Comment by devotox** [comment]
Ok so I have traced this deeper and it involves the use of a `.yarnclean` file. 

When a yarn install is done this will delete every file or folder in that file and I had .travis.yml as one of the files cleaned from node_modules. 

Because `ember-cli/utilities/load-config` uses the `find-up` module this had previously been ok as it just iterates all the way up to the main application. Now that I removed the last `.travis.yml` it could possibly find the issue then came to light. This i guess is not technically a bug as `ember-cli` expects to have its own module to include this file. 

This does raise the point though about whether that is the best way to figure out its compatible node versions?
